core/scss/components/alert/_alert.scss

```diff
@@ -7,18 +7,56 @@
 // of a status, or for displaying a validation message that
 // alerts someone of an important piece of information.
 //
-// .su-alert-success  - Successful Transaction Message
-// .su-alert-warning  - Warning Message
-// .su-alert-error    - Error Message
-// .su-alert-info     - Informative Message
+// .su-alert--success  - Successful Transaction Message
+// .su-alert--warning  - Warning Message
+// .su-alert--error    - Error Message
+// .su-alert--info     - Informative Message
 //
 // Markup: ../templates/components/alert/alert.twig
 //
 // Style guide: Components.Alert
 //
 .su-alert {
-  @include alerts;
-}
+  @include margin(1.5em null null);
```

Comment:
I only have one question - is there a reason to add a top margin for this component? We don't have margins for cards or hero block for example. Everything else looks great to me!

Reply:
Oh good catch!

@josephgknox What gives?

Reply:
Just keeping you both on your toes. :)

Really, this shouldn't be there, as this type of spacing/margin should be applied more globally (theme level, project level, etc.) rather than on an individual component.

Thanks for catching this. I've pushed up a fix.